llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -126,6 +126,11 @@ static cl::opt<bool> MulConstantOptimization(
              "SHIFT, LEA, etc."),
     cl::Hidden);
 
+static cl::opt<bool>
+    EnableBaseIndexUpdate("update-baseIndex", cl::init(true),
+                          cl::desc("Update the value of base and index"),
```

Comment:
Name and description don't mean anything standalone

Reply:
Sure, I will update the information